### DISCUSSION TITLE
How to config the server address of the Mac/Windows client?

### DISCUSSION THREAD

**Discussion by woostundy** [Postcatlab/postcat#228] [Q&A]
I didn't find any setting to change the server address, did I miss some doc?

怎么修改Mac客户端的服务地址呢？我没找到配置的地方，也没看到相关文档。

**Comment by scarqin** [comment]
@woostundy Do you mean self-host server address？

**Reply by woostundy** [reply]
@scarqin  Yes. Is this supported?

**Reply by scarqin** [reply]
@woostundy Eoapi supports but Postcat currently does not support,check this issue: https://github.com/eolinker/postcat/issues/215
